datadog-profiling/src/collections/string_table/mod.rs

```diff
@@ -269,7 +269,7 @@ mod tests {
                     // on the list.  We can't look inside the `StringTable`
                     // in a non-desctrive way, but fortunately we have the
                     // `golden_list` to compare against.
-                    assert_eq!(string, golden_list[str_id.to_offset()]);
+                    assert_eq!(string, golden_list[usize::from(str_id)]);
```

Comment:
Why the change?

Reply:
The trait that `to_offset` is from is gone.